bluebottle/bb_projects/models.py

```diff
@@ -236,6 +235,10 @@ def people_needed(self):
 
         return requested - counts['total'] + (counts['externals'] or 0)
 
+    @property
+    def account_bic(self):
+        return self.account_details
+
```

Comment:
Jeez-Louis, I should not do reviews after such a long day. Your failing test prove I might be right! 
I created a PR against this which fixes it.
Nice work @ashutoshb. I like how you thought about keeping this compatible with current Reef!